modules/administration-guide/pages/applying-the-bitbucket-application-link-secret.adoc

```diff
@@ -0,0 +1,58 @@
+:_content-type: PROCEDURE
+:description: Applying the Bitbucket application link Secret
+:keywords: applying-bitbucket, apply-the-bitbucket, apply-bitbucket, apply-secret, applying-secret, apply-a-secret, applying-a-secret, bitbucket, bitbucket-application, bitbucket-app, bitbucket-application-link, bitbucket-app-link, bitbucket-secret, secret
+:navtitle: Applying the Bitbucket application link Secret
+// :page-aliases:
+
+[id="applying-the-bitbucket-application-link-secret_{context}"]
+= Applying the Bitbucket application link Secret
+
+Enable the users of your {prod-short} instance to access a private Git repository that is hosted using Bitbucket:
+
+.Prerequisites
+* The repository owner has xref:user-guide:setting-up-the-bitbucket-application-link.adoc[set up the Bitbucket application link].
+* The repository owner has given you the (1) *privatepkcs8-stripped.pem* file and (2) *bitbucket_consumer_key* file.
+* `{orch-cli}` is installed in the operating system you are using.
+////
+{orch-cli}=oc
+https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli
+https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/
+////
+
+.Procedure
+
+. Prepare the secret as follows:
++
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+kind: Secret
+apiVersion: v1
+metadata:
+  name: bitbucket-oauth-config
+  namespace: {prod-namespace} <1>
+  labels:
+    app.kubernetes.io/component: oauth-scm-configuration
+    app.kubernetes.io/part-of: che.eclipse.org
+  annotations:
+    che.eclipse.org/oauth-scm-server: bitbucket
+    che.eclipse.org/scm-server-endpoint: __<bitbucket_server_url>__ <2>
+type: Opaque
+data:
+  private.key: __<base64_content_of_privatepkcs8-stripped.pem>__ <3>
+  consumer.key: __<base64_content_of_bitbucket_server_consumer_key>__ <4>
```

Comment:
```suggestion
  consumer.key: __<base64_content_of_bitbucket_server_consumer_key>__ <4>
  shared_secret: __<base64_content_of_bitbucket-shared-secret>__ <5>
```

To match: https://github.com/eclipse-che/blog/blob/95b152e3e7c0434b72bf4d5bf511e081b81d788c/_posts/2022-XX-XX-configuring-git-repositories.adoc#bitbucket

Reply:
Thank you. Next time I'll be more careful with the last line.